src/Home/Home.tsx

```diff
@@ -118,7 +123,24 @@ const HomeScreen: FunctionComponent = () => {
 
   const bottomContainerStyle = {
     ...style.bottomContainer,
-    maxHeight: insets.bottom + Layout.screenHeight * 0.475,
+    maxHeight: Platform.select({
```

Comment:
It seems like there are a bunch of "magic numbers" here, but this is the only way I can think off to make this design work. It gets really complex when you're working with different screen sizes and supporting iOS + Android, plus we have this background image, absolutely positioned elements, etc.

Reply:
one thing you can consider is using named constants. for example:
```typescript
const maxHeightIOS = insets.bottom + Layout.screenHeight * 0.475
const maxHeightAndroid = insets.bottom + Layout.screenHeight * 0.475 - 50
```

while still magic number-ish you have at least provided the reader a bit of context on the reasoning.

Reply:
Good call!